browser/ui/webui/brave_federated/federated_internals_page_handler.cc

```diff
@@ -27,32 +29,39 @@ FederatedInternalsPageHandler::FederatedInternalsPageHandler(
               profile)
               ->GetDataStoreService()) {}
 
-FederatedInternalsPageHandler::~FederatedInternalsPageHandler() {}
+FederatedInternalsPageHandler::~FederatedInternalsPageHandler() = default;
 
-void FederatedInternalsPageHandler::GetAdStoreInfo() {
+void FederatedInternalsPageHandler::GetDataStoreInfo() {
   if (!data_store_service_)
     return;
-  auto* const ad_notification_data_store =
-      data_store_service_->GetAdNotificationTimingDataStore();
+  const auto* ad_notification_data_store = data_store_service_->GetDataStore(
+      brave_federated::kAdNotificationTaskName);
 
-  ad_notification_data_store->LoadLogs(
-      base::BindOnce(&FederatedInternalsPageHandler::OnAdStoreInfoAvailable,
+  if (!ad_notification_data_store) {
+    return;
+  }
+
+  ad_notification_data_store->LoadTrainingData(
+      base::BindOnce(&FederatedInternalsPageHandler::OnDataStoreInfoAvailable,
                      weak_ptr_factory_.GetWeakPtr()));
 }
 
-void FederatedInternalsPageHandler::OnAdStoreInfoAvailable(
-    brave_federated::AdNotificationTimingDataStore::
-        IdToAdNotificationTimingTaskLogMap ad_notification_timing_logs) {
-  std::vector<federated_internals::mojom::AdStoreLogPtr> ad_logs;
-  for (auto const& object : ad_notification_timing_logs) {
-    auto ad_store_log = federated_internals::mojom::AdStoreLog::New();
-    const auto& log = object.second;
-    ad_store_log->log_id = log.id;
-    ad_store_log->log_time = log.time.ToJsTime();
-    ad_store_log->log_locale = log.locale;
-    ad_store_log->log_number_of_tabs = log.number_of_tabs;
-    ad_store_log->log_label = log.label;
-    ad_logs.emplace_back(std::move(ad_store_log));
+void FederatedInternalsPageHandler::OnDataStoreInfoAvailable(
+    brave_federated::DataStore::TrainingData training_data) {
+  std::vector<federated_internals::mojom::InternalsTrainingInstancePtr>
+      internals_training_instances;
+  for (const auto& training_instance : training_data) {
+    auto internals_training_instance =
+        federated_internals::mojom::TrainingInstance::New();
+    for (const auto& covariate : training_instance.second) {
```

Comment:
Maybe assign `training_instance.second` to a variable named `covariates` to make the code clearer